content/manager/security.md

```diff
@@ -4,436 +4,701 @@ title: Security
+- `audit_log_file` - sets the full path to the auditing file on the manager
+- `audit_log_level` - modifying the log level will produce less or more elaborate security auditing; valid values are:
 CRITICAL, ERROR, WARNING, INFO or DEBUG.
-- audit_log_file_size_MB - limits the log file size. By default, the file is limited to 100 MB. When the file reaches
-that size, it will be renamed with the extension ".1" and a new log file will be created.
-- audit_log_files_backup_count - sets the maximum number of old log files to keep. By default this value is set to 20.
+- `audit_log_file_size_MB` - limits the log file size. By default, the file is limited to 100 MB. When the file reaches
+that size, it will be renamed with the extension ".1" and a new log file will be created (older files will be renamed
+with the extension ".2", ".3" and so forth).
+- `audit_log_files_backup_count` - sets the maximum number of old log files to keep. By default this value is set to 20.
 That means that up to 20 old log files can be created, after which the oldest file will be removed.
 
-
-# Clients
+<br>
+## Clients
 
 Different clients can be used to send requests to the REST service, and all go through an authentication process.
 Here we review what each client requires in order to send a secured request.
 
+### CLI - cfy commands
+#### credentials
+{{% gsNote title="User credentials must be set before bootstrapping" %}}{{% /gsNote %}}
 
-## Web UI
-Login - with username and password
-
-{{% gsNote title="Note" %}}
-This client only supports authentication with username-password.
-{{% /gsNote %}}
-
-{{% gsNote title="Note" %}}
-Available in the premium edition only
-{{% /gsNote %}}
-
-## Cloudify CLI: cfy commands
-cfy commands that connect to the REST service (e.g. "deployments create") send username and password in a header on each
-request. Setting these values is done once, via these environment variables:
-
- - CLOUDIFY_USERNAME
- - CLOUDIFY_PASSWORD
+Cfy commands automatically add the "Authorization" header to each request sent to the manager, containing the username
+and password of the current user. Setting the user credentials is done once, by exporting these environment variables:
 
-### SSL CLI configuration
+ - export CLOUDIFY_USERNAME=my_username
+ - export CLOUDIFY_PASSWORD=my_password
 
-#### Connect with a manager secured by SSL:
-  - After bootstrap, every request to the secured manager will be on port 443, using the 'https' protocol.
-  - The `use` command requires specifing the port option to be 443
-      <br>e.g.
-      {{< gsHighlight  bash  >}}
-        'use -t <manager-ip-address> --port 443'
-      {{< /gsHighlight >}}
+<br>
+#### Client SSL configuration
+**Connecting to the manager over SSL**<br>
+After bootstrap, requests sent to the manager must use port 443 and the 'https' protocol. <br>
+The `cfy use` command requires specifying the port option to be 443:
```

Comment:
It may be worth mentioning that `cfy use` is not required after bootstrap from the client that initiated the bootstrap. it is only required when using a different client (or running cfy from a different location that doesn't already include the context)